docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -1275,6 +1275,21 @@ The Tooltip's prop `componentsProps` was deprecated in favor of `slotProps`:
  />
 ```
 
+## Typography
+
+There is no codemod to manage this change.
+
+### paragraph
+
+The Typography's prop `paragraph` was deprecated in favor of setting `margin-bottom` using the sx props:
+
+```diff
+ <Typography
+-  paragraph
++  sx={{ marginBottom: "16px" }}
```

Comment:
I think the most important bit is mentioning the `component` prop, as you did in the prop type comments.

```suggestion
The Typography's `paragraph` prop was deprecated in favor of using the `component` prop. The `16px` of margin-bottom that was automatically added to the element when using `paragraph` must be manually handled now.

```diff
 <Typography
-  paragraph
+  component="p"
```